### PR TITLE
Remove references to sawtooth cluster from docs

### PR DESCRIPTION
**Pull request by ojalatodd**
Removed references to 'sawtooth cluster' from docs,
since the cluster command will be removed.

### CODE REVIEW

File: docs/source/app_developers_guide/environment_setup.rst [hunk 1]
```diff
@@ -30,54 +30,31 @@ The following tools are required:
 * `Docker Compose <https://docs.docker.com/compose/install/>`_ (Linux only)
 
 
-Step One: Clone Repository
---------------------------
+Get the Docker Yaml File
```
Comment (vaporos) [inline docs/source/app_developers_guide/environment_setup.rst]: Suggest "Download the Docker Compose YAML File"

File: docs/Makefile [hunk 2]
```diff
@@ -64,6 +64,7 @@ html: templates cli
 	$(SPHINXBUILD) -b html $(ALLSPHINXOPTS) $(BUILDDIR)/html
 	@echo
 	@echo "Build finished. The HTML pages are in $(BUILDDIR)/html."
+	@cp ../docker/compose/sawtooth-demo.yaml $(BUILDDIR)/html/app_developers_guide/sawtooth-demo.yaml
```
Comment (vaporos) [inline docs/Makefile]: In the documentation below, it uses 'sawtooth-default.yaml' not 'sawtooth-demo.yaml', so the last argument here should probably be '$(BUILDDIR)/html/app_developers_guide/sawtooth-default.yaml'

**Comment by vaporos** [comment]
STL-92

File: docs/source/app_developers_guide/environment_setup.rst (hunk 1 above)
Comment (ojalatodd) [inline docs/source/app_developers_guide/environment_setup.rst]: This is being changed by another PR. I'll rebase this one when that gets merged today.

File: docs/Makefile (hunk 2 above)
Comment (ojalatodd) [inline docs/Makefile]: This was also changed today in the other PR. I'll rebase once it's merged.

**Pull request merged**